SPECS/r8169-module-alt.spec

```diff
@@ -0,0 +1,57 @@
+%define vendor_name Realtek
+%define vendor_label realtek
+%define driver_name r8169
+
+%if %undefined module_dir
+%define module_dir override
+%endif
+
+Summary: %{vendor_name} %{driver_name} device drivers
+Name: %{driver_name}-module-alt
+Version: 4.19.128
+Release: 1%{?dist}
+License: GPL
+
+# Source taken from https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/drivers/net/ethernet/realtek?h=v4.19.128
+Source0: %{driver_name}-%{version}.tar.gz
+
+BuildRequires: gcc
+BuildRequires: kernel-devel
+Provides: vendor-driver
+Requires: kernel-uname-r = %{kernel_version}
+Requires(post): /usr/sbin/depmod
+Requires(postun): /usr/sbin/depmod
+
+%description
+%{vendor_name} %{driver_name} device drivers for the Linux Kernel
+version %{kernel_version}.
+
+%prep
+%autosetup -p1 -n %{driver_name}-%{version}
+
+%build
+%{?cov_wrap} %{make_build} -C /lib/modules/%{kernel_version}/build M=$(pwd) KSRC=/lib/modules/%{kernel_version}/build modules
+
+%install
+%{?cov_wrap} %{__make} %{?_smp_mflags} -C /lib/modules/%{kernel_version}/build M=$(pwd) INSTALL_MOD_PATH=%{buildroot} INSTALL_MOD_DIR=%{module_dir} DEPMOD=/bin/true modules_install
```

Comment:
idem. And is the `_smp_mflags` macro available in our build environment or is it a Citrix thing?

Reply:
It is available and necessary to have parallel build.
```
# rpm --eval "Eval %{?_smp_mflags}"
Eval -j12
```